app/src/main/java/com/framgia/forder/data/source/remote/api/response/DomainToRequestShopResponse.java

```diff
@@ -0,0 +1,115 @@
+package com.framgia.forder.data.source.remote.api.response;
+
+import com.framgia.forder.data.model.DomainManagement;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.DomainManagement. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports